attestation-evidence-broker/docs/design/design.md

```diff
@@ -0,0 +1,121 @@
+This document provides that how to use Attestation Evidence Broker (AEB) which is a host service to satisfy the requests from guest for getting **SEV(-ES) runtime remote attestation report**.
+
+# Background of SEV(-ES) runtime remote attestation
+
+Compared with SEV SNP and Intel TDX, the runtime remote attestation of SEV(-ES) does NOT support guest-provided data to be included into the attestation report [1,2]. In addition, SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values. Although SEV(-ES) has this limitation, runtime remote attestation is still useful. When the SEV(-ES) guest is running, it can report the attestation evidence with fixed launch digest as a heartbeat for trusted healthy check.
```

Comment:
s/runtime remote attestation/runtime attestation/

Reply:
s/guest-provided/user-defined/

Reply:
s/to be included into/included in/

Reply:
s/In addition, SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values./In addition, SEV(-ES) does NOT support dynamic measurement. During runtime, it can only generate static attestation report with the constant launch digest.

Reply:
The statement about `guest-provided data to be included into the attestation report` is refered in [amd SNP spec](https://www.amd.com/system/files/TechDocs/56860.pdf).
![image](https://user-images.githubusercontent.com/26923638/148215028-6399fcc8-8c7a-4fb3-bd8b-1e6a20e0fde6.png)

Reply:
OK. Use gurst-provided instead of user-defined.